spec.bs

```diff
@@ -111,7 +121,7 @@ dictionary AuctionAdInterestGroup {
 The <dfn for=Navigator method>joinAdInterestGroup(|group|)</dfn> method steps are:
 <div class="note">
 
-Temporarily, Chromium does not include the <span class="allow-2119">[=dictionary member/required=]</span> keyword
+Temporarily, Chromium does not include the <span class="allow-2119">required</span> keyword

```

Comment:
it worked until yesterday, but started conplaining about the 2119 error again locally. Did bikeshed update but still having the error.
Will try changing it back later.